ephios/core/ical.py

```diff
@@ -58,9 +58,21 @@ def item_start_datetime(self, item):
     def item_end_datetime(self, item):
         return item.participations.all()[0].end_time
 
+    def item_status(self, item):
+        # The ical status field can be CONFIRMED|TENTATIVE|CANCELLED
+        return {
+            AbstractParticipation.States.CONFIRMED: "CONFIRMED",
+            AbstractParticipation.States.REQUESTED: "TENTATIVE",  # displayed less opaque in many calendars
+            AbstractParticipation.States.RESPONSIBLE_REJECTED: "CANCELLED",  # often displayed struck out
```

Comment:
do we really want to display these as well? usually there is no way of editing imported ICS feeds, and I personally hate it when I have events in my calender that I will not attend

Reply:
Outlook removes the event completely when you cancel an invitation/event. Which clients did you test?

Reply:
![image](https://github.com/user-attachments/assets/51dfa6af-0b5d-4be2-a128-13a516dc7e0f)
This is the nextcloud calendar.

----

Google maps doesn't do anything to it:
![image](https://github.com/user-attachments/assets/c9662bd2-1baf-47a7-8bb5-beab1839b343)

----

Thunderbird is like nextcloud:
![image](https://github.com/user-attachments/assets/25dd88d4-d2db-4750-8973-fa94f585c7a5)

---
My FOSS calendar app doesn't display anything either:
![image](https://github.com/user-attachments/assets/9d8bb5cc-599e-4fd2-8fcb-e72e2137543d)